src/ring_theory/roots_of_unity.lean

```diff
@@ -746,6 +746,25 @@ begin
   int.cast_one, aeval_one, alg_hom.map_sub, sub_self]
 end
 
+/-- The reduction modulo `p` of the minimal polynomial of `μ` is separable. -/
+lemma minimal_polynomial_is_separable {p : ℕ} [hprime : fact p.prime] (hdiv : ¬p ∣ n) : separable
```

Comment:
Should this be called something like `separable_minimal_polynomial_mod`?
And similar for the lemma below...